src/celscript/lua/celx_phase.cpp

```diff
@@ -160,10 +157,7 @@ static int phase_getorientation(lua_State* l)
     auto phase = this_phase(l);
 
     double tdb = celx.safeGetNumber(2, WrongType, "Argument to phase:getorientation() must be number", 0.0);
-    if (tdb < phase->startTime())
-        tdb = phase->startTime();
-    else if (tdb > phase->endTime())
-        tdb = phase->endTime();
+    tdb = phase->clamp(tdb);
```

Comment:
clamp to [0, 1]?

Reply:
this is to clamp between start time and end time of a phase.